lib/grizzly/zwave/command_classes/network_management_installation_maintenance.ex

```diff
@@ -83,7 +83,7 @@ defmodule Grizzly.ZWave.CommandClasses.NetworkManagementInstallationMaintenance
   def rssi_to_byte(:rssi_below_sensitivity), do: 0x7D
   def rssi_to_byte(:rssi_max_power_saturated), do: 0x7E
   def rssi_to_byte(:rssi_not_available), do: 0x7F
-  def rssi_to_byte(value) when value in -94..-32, do: 256 + value
+  def rssi_to_byte(value) when value in -128..-32, do: 256 + value
```

Comment:
The `rssi` spec still says -94 for the low number rather than -128 at the top of the file.

Reply:
Good catch! Fixed.